Tvl.VisualStudio.JustMyCodeToggle/Tvl.VisualStudio.JustMyCodeToggle.csproj

```diff
@@ -1,94 +1,102 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project>
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
-
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This change moves from the new project system to the old/legacy project system, so it would be preferable to avoid that.

Reply:
That is odd, I took the project template from the latest VS2022 beta for a VSIX plugin, i went ahead and manually updated the project file.